repos/bluerobotics/cockpit/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Cockpit",
+    "website": "https://github.com/bluerobotics/cockpit",
+    "docker": "patrickelectric/cockpit:latest",
```

Comment:
The tags are fetched automatically by CI. 

```suggestion
    "docker": "patrickelectric/cockpit",
```

Reply:
Done.